.circleci/config.yml

```diff
@@ -91,6 +91,7 @@ aliases:
       export PERL5OPT="$COVEROPT,-db,cover_db_${CIRCLE_JOB}"
       echo PERL5OPT="$PERL5OPT"
       make test-$CIRCLE_JOB
+    no_output_timeout: 30m
```

Comment:
I'm not sure, but considering https://progress.opensuse.org/issues/66664 I would say that extending the deadline only means it would take longer for us to find out about the errors.

Reply:
I know. That's what I said in the same ticket. But I see no harm in testing it out anyway. :-)